service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storage/CreateAzureStorageStepTest.java

```diff
@@ -43,7 +43,7 @@ public class CreateAzureStorageStepTest extends BaseAzureTest {
   @Mock private AzureConfiguration mockAzureConfig;
   @Mock private AzureCloudContext mockAzureCloudContext;
   @Mock private StorageManager mockStorageManager;
-  @Mock private ManagementException mockException;
+  private ManagementException resourceNotFoundException;
```

Comment:
It seems like the new version of Logback doesn't like the way we previously logged this mock's fields, which led to `createStorage_failsToCreate` throwing an NPE inside the step code.

I'm also not sure why we needed to mock out an exception in the first place, though, so I replaced it with a real object here.